### PR TITLE
Create less stuff when systemd-tmpfiles --create is executed (during installation or otherwise)

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: tmpfiles.d/etc.conf.m4 [hunk 1]
```diff
@@ -15,8 +15,8 @@ t /etc/mtab - - - - security.SMACK64=_
...
 L! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
 )m4_dnl
-C /etc/nsswitch.conf - - - -
+C! /etc/nsswitch.conf - - - -
 m4_ifdef(`HAVE_PAM',
-C /etc/pam.d - - - -
+C! /etc/pam.d - - - -
 )m4_dnl
-C /etc/issue - - - -
+C! /etc/issue - - - -
```
Comment (poettering) [inline tmpfiles.d/etc.conf.m4:22]: hmm, while we are at it, i wonder if we should switch these all to be symlinks, i.e. just replace C by L. The effect would be the same I guess, but we would benefit from vendor updates in /usr...

And the L stuff also has the factory magic built-in too...

**Comment by poettering** [comment]
looks good to me, but maybe let's also do the C → L thing?

**Comment by poettering** [comment]
(btw, i don't think we should do the C → L transition anymore, because that takes the chance for us away to empty the factory dir sooner or later, as it means we have to keep all those files in there, whatever we do, since systems might link to those files. But as software might get fixed to not require the files in /etc anymore we can't drop them from the factory dirs, unfortunately. So let's continue to copy them, then)

**Comment by keszybz** [comment]
I was going to say: the C → L transition is risky because people might put dropins in there, and/or edit the files with automatic scripts.

**Pull request merged**